include/bus_handler.hpp

```diff
@@ -65,6 +65,10 @@ class BusHandler
                                      }
                                      return 0;
                                  });
+        iface->register_method("ExecuteFunction",
```

Comment:
do we need to catch exceptions ?

Reply:
currently am catching the exception and returning false.